app/javascript/.storybook/main.js

```diff
@@ -86,6 +86,25 @@ module.exports = {
   },
   babel: async (options) => ({
     ...options,
-    plugins: [...options.plugins, 'inline-react-svg'],
+    plugins: [
```

Comment:
This configuration is set in Storbook's config as well as in Babel's config down below.